lib/node_modules/@stdlib/math/base/special/hyp2f1/lib/main.js

```diff
@@ -0,0 +1,256 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*
+* ## Notice
+*
+* The original C++ code and copyright notice are from the [SciPy library]{@link https://github.com/scipy/xsf/blob/main/include/xsf/cephes/hyp2f1.h}. The implementation has been modified for JavaScript.
+*
+* ```text
+* (C) Copyright SciPy Developers 2024.
+*
+* Use, modification and distribution are subject to the
+* SciPy Software License. (See accompanying file
+* LICENSE or copy at https://github.com/scipy/scipy/blob/main/LICENSE.txt)
+* ```
+*/
+
+'use strict';
+
+// MODULES //
+
+var isnan = require( '@stdlib/assert/is-nan' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var round = require( '@stdlib/math/base/special/round' );
+var gamma = require( '@stdlib/math/base/special/gamma' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var isNonPositiveInteger = require( './utils.js' ).isNonPositiveInteger;
+var isInteger = require( './utils.js' ).isInteger;
+var hys2f1 = require( './hys2f1.js' );
+var hyt2f1 = require( './hyt2f1.js' );
+
+
+// VARIABLES //
+
+var ETHRESH = 1.0e-12;
```

Comment:
For these constants which get repeated, I'd move all of them to a JSON file (e.g., `./config.json`) and then import them as needed. E.g.,

```
var ETHRESH = config.ETHRESH;
```